test/integration/operator_test.go

```diff
@@ -29,14 +32,13 @@ const (
 )
 
 const (
-	kacName     = "kubearchive"
-	a13eName    = kacName + "-a13e"
-	sinkName    = kacName + "-sink"
+	a13eName    = test.KACName + "-a13e"
+	sinkName    = test.KACName + "-sink"
 	filtersName = "sink-filters"
 )
 
 func TestEmptyKAC(t *testing.T) {
-	runTest(t, emptyKAC, map[string]any{})
+	runTest(t, "emptyKAC", map[string]any{})
```

Comment:
Is there any reason to use this approach for running different versions of a test instead of the one that we use in the unit tests with [a map to store the different test cases](https://go.dev/wiki/TableDrivenTests#using-a-map-to-store-test-cases)?

I think it's the standard way to put different use cases of the same "feature" and it should be a small change from what you have right now.

Reply:
@maruiz93, the change is made. Can you review again?

Reply:
lgtm!